src/widgets/pixbuf_loader.rs

```diff
@@ -18,16 +18,12 @@ pub struct PixbufLoader {
 impl PixbufLoader {
     /// Creates a new pixbuf loader object.
     ///
-    /// # Failures
-    /// Returns `None` if the pixbuf loader cannot be created.
-    pub fn new() -> Option<PixbufLoader> {
+    /// # Panics
+    /// Fails if the pixbuf loader cannot be retrieved. 
+    pub fn new() -> PixbufLoader {
```

Comment:
All at once, or at least at once by library. If we do only a part at each time, we could forgot some and it would take a much longer time. So let's open an issue for each concerned library. After that, I'll put all rust-gnome libraries on crates.io (I'll do pango).